### DISCUSSION TITLE
URLを貼ってもサムネイルが表示されない。

### DISCUSSION THREAD

**Discussion by ktst79** [increments/qiita-discussions#934] [要望・アイデア・不具合]
Qiita記事にURLを貼ってもサムネイルが表示されない事象が発生しています。
https://qiita.com/ktst79/items/c1ddf33e18e9cfeba893

![image](https://github.com/user-attachments/assets/01e1944e-a634-4391-b4fe-f60921cc640b)

以下の問題と同件だとするとサムネイルのキャッシュの問題かもしれません。
https://github.com/increments/qiita-discussions/discussions/921

**Comment by tanaka-asami415** [comment]
@ktst79 
ご不便をおかけして、申し訳ございません。

ご指摘の通り、同様にキャッシュの問題かと思われます。
弊社でキャッシュクリアをしましたので、ご確認をいただけますと幸いです🙇

**Reply by ktst79** [reply]
ありがとうございます。問題が解決していることを確認しました。
